pages/ksphere/konvoy/1.4/release-notes/open-source-attribution/index.md

```diff
@@ -114,14 +114,3 @@ The table below lists the third party open source software which is provided by
 |utils                      |v0.0.0-20191010214722-8d271d903fe4  |[Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                                 |
 |controller-runtime         |v0.3.1-0.20191022174215-ad57a976ffa1|[Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                                 |
 |yaml                       |v1.1.0                              |[MIT License](https://github.com/weapp/boto/blob/master/LICENSE.txt)                                              |
-
-
```

Comment:
Still concerned about this removal. Wondering if this didn't get backported?

Reply:
this section doesn't exist in master or in 1.5 docs. See https://docs.d2iq.com/ksphere/konvoy/1.5/release-notes/open-source-attribution/

This removal matches the latest docs. But if you look [at this line](https://github.com/mesosphere/dcos-docs-site/blame/8e7706bf382a79c5dfc190afb25f85e84b5cc487/pages/ksphere/konvoy/1.4/release-notes/open-source-attribution/index.md#L57) we're still attributing that library to Mozilla license.

Reply:
Alright, I talked to Senthil and yes the info that is in 1.4 oss topic IS needed and was inadvertently removed, but probably not by this PR specifically. I am opening a new PR on Konvoy to fix this.